test_sum_terms.py

```diff
@@ -0,0 +1,8 @@
+INVALID_INPUT = u'the argument must be a positive whole number or 0'
+Test.assert_equals(series_sum(0), "0.00")
```

Comment:
Your tests can't be passing **on your machine**. When I run your tests on my machine I get the following:

```
______________________ ERROR collecting test_sum_terms.py ______________________
test_sum_terms.py:2: in <module>
    Test.assert_equals(series_sum(0), "0.00")
E   NameError: name 'Test' is not defined
=========================== 1 error in 0.05 seconds ============================
```